spread.yaml

```diff
@@ -838,7 +838,45 @@ backends:
             - ubuntu-22.04-arm64:
                   username: ubuntu
                   password: ubuntu
-            # Debian
+            # Debian Forky
+            - debian-14-amd64:
+                  username: '$(HOST: echo "$AUTOPKGTEST_NORMAL_USER")'
+                  password: '$(HOST: echo "$AUTOPKGTEST_NORMAL_USER")'
+            - debian-14-i386:
```

Comment:
do we need i386?

Reply:
That's a good question. I think it's dead now but I will confirm before removal.